From 8e119eb32e01066c23c3d96bd2b42032e03f7628 Mon Sep 17 00:00:00 2001
From: Yang Rong <rong.r.yang@intel.com>
Date: Thu, 13 Apr 2017 19:32:51 +0800
Subject: utest: fix image qualifier of compiler_fill_gl_image test.

After clang check the image qualifier, can't use default qualifier
to write_image.

Signed-off-by: Yang Rong <rong.r.yang@intel.com>
Reviewed-by: Pan Xiuli <xiuli.pan@intel.com>

diff --git a/kernels/test_fill_gl_image.cl b/kernels/test_fill_gl_image.cl
index 4250a57..7b5dce7 100644
--- a/kernels/test_fill_gl_image.cl
+++ b/kernels/test_fill_gl_image.cl
@@ -1,5 +1,5 @@
 __kernel void
-test_fill_gl_image(image2d_t img, int color)
+test_fill_gl_image(write_only image2d_t img, int color)
 {
 	int2 coord;
         float4 color_v4;
-- 
cgit v0.10.2

